src/webgpu/api/validation/encoding/cmds/render/draw.spec.ts

```diff
@@ -723,3 +723,24 @@ In this test we test that only the last setting for a buffer slot take account.
 `
   )
   .unimplemented();
+
+g.test(`max_draw_count`)
+  .desc(
+    `
+In this test we test that draw count which exceeds GPURenderPassDescriptor.maxDrawCount
+causes validation error on GPUCommandEncoder.finish(). The test sets specified maxDrawCount,
+calls specified draw call specified times, and checks whether GPUCommandEncoder.finish()
+causes a validation error.
+    - x= all draw types
+    - draw count {0, 1, 2, 10}
+    - max draw count {0, 1, 2, 10}
+TODO: Missing test case Bundle
```

Comment:
If this will be part of this test (I think it should be), maybe you can go ahead and propose how it'll be done here. Maybe simple a boolean `useBundles` which splits the calls between, say, 2 bundles. Or `bundleFirstHalf` and `bundleSecondHalf` booleans perhaps.

Reply:
Thanks for the suggestion. I want to take `bundleFirst/SecondHalf` idea because I feel I want to add bundle + non-bundle combination test.

Reply:
Updated, thanks.